Common/src/interpolation_structure.cpp

```diff
@@ -1028,61 +1106,89 @@ void CMirror::Set_TransferCoeff(CConfig **config) {
 

```

Comment:
Isoparametric is currently working for sliding meshes, but I never tested CMirror, though I expect it to work properly. Do you have any test case I can use to check it?